pyqt-apps/siriushla/as_ap_sofb/ioc_control/main.py

```diff
@@ -444,3 +457,94 @@ def __init__(self, parent, prefix, ctrls, acc='SI'):
+        lbl_tout = QLabel('Timeout', self, alignment=Qt.AlignCenter)
+        lbl_diff = QLabel('Diff', self, alignment=Qt.AlignCenter)
+        ld_ok = SiriusLabel(self, lamb('LoopPerfItersOk-Mon'))
+        ld_tout = SiriusLabel(self, lamb('LoopPerfItersTOut-Mon'))
+        ld_diff = SiriusLabel(self, lamb('LoopPerfItersDiff-Mon'))
+        lay_iters = QGridLayout()
+        lay_iters.addWidget(lbl_iters, 1, 0)
+        lay_iters.addWidget(lbl_ok, 0, 1)
+        lay_iters.addWidget(lbl_tout, 0, 2)
+        lay_iters.addWidget(lbl_diff, 0, 3)
+        lay_iters.addWidget(ld_ok, 1, 1, alignment=Qt.AlignCenter)
+        lay_iters.addWidget(ld_tout, 1, 2, alignment=Qt.AlignCenter)
+        lay_iters.addWidget(ld_diff, 1, 3, alignment=Qt.AlignCenter)
+
+        lbl_psd = QLabel('# of PSs with Diffs:', self)
+        lbl_avg = QLabel('AVG', self, alignment=Qt.AlignCenter)
+        lbl_std = QLabel('STD', self, alignment=Qt.AlignCenter)
+        lbl_max = QLabel('MAX', self, alignment=Qt.AlignCenter)
+        ld_avg = SiriusLabel(self, lamb('LoopPerfDiffNrPSMax-Mon'))
+        ld_std = SiriusLabel(self, lamb('LoopPerfDiffNrPSAvg-Mon'))
+        ld_max = SiriusLabel(self, lamb('LoopPerfDiffNrPSStd-Mon'))
+        lay_psd = QGridLayout()
+        lay_psd.addWidget(lbl_psd, 1, 0)
+        lay_psd.addWidget(lbl_avg, 0, 1)
+        lay_psd.addWidget(lbl_std, 0, 2)
+        lay_psd.addWidget(lbl_max, 0, 3)
+        lay_psd.addWidget(ld_avg, 1, 1, alignment=Qt.AlignCenter)
+        lay_psd.addWidget(ld_std, 1, 2, alignment=Qt.AlignCenter)
+        lay_psd.addWidget(ld_max, 1, 3, alignment=Qt.AlignCenter)
+
+        lbl_tim = QLabel('Duration Statistics', self, alignment=Qt.AlignCenter)
+        lbl_unt = QLabel('[ms]', self, alignment=Qt.AlignCenter)
+        lbl_avg = QLabel('AVG', self, alignment=Qt.AlignCenter)
+        lbl_std = QLabel('STD', self, alignment=Qt.AlignCenter)
+        lbl_min = QLabel('MIN', self, alignment=Qt.AlignCenter)
+        lbl_max = QLabel('MAX', self, alignment=Qt.AlignCenter)
+        lbl_avg.setStyleSheet('min-width: 5em;')
+        lbl_std.setStyleSheet('min-width: 5em;')
+        lbl_min.setStyleSheet('min-width: 5em;')
+        lbl_max.setStyleSheet('min-width: 5em;')
+        lay_tim = QGridLayout()
+        lay_tim.addWidget(lbl_tim, 0, 0, 1, 5)
+        lay_tim.addWidget(lbl_unt, 1, 0)
+        lay_tim.addWidget(lbl_avg, 1, 1)
+        lay_tim.addWidget(lbl_std, 1, 2)
+        lay_tim.addWidget(lbl_min, 1, 3)
+        lay_tim.addWidget(lbl_max, 1, 4)
+
+        labs = [
+            'Get Orbit', 'Get Kick', 'Calculate', 'Process', 'Apply', 'Total']
+        nms = ['GetO', 'GetK', 'Calc', 'Proc', 'App', 'Tot']
+        for i, (nm, lab) in enumerate(zip(nms, labs)):
+            lbl = QLabel(lab, self)
+            lay_tim.addWidget(lbl, i+2, 0)
+            for j, ld in enumerate(['Avg', 'Std', 'Min', 'Max']):
+                ld_ = SiriusLabel(
+                    self, lamb(f'LoopPerfTim{lab:s}{ld:s}-Mon'))
```

Comment:
`labs` have some strings with spaces, I think the spaces have to be removed here

Reply:
right! i was supposed to be `nm` not `lab` there! Thanks!

Reply:
done!